src/screens/Lessons/screens/Lesson/index.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react'
-import {lessonTitleText} from 'utils/text'
 import Screen from 'components/Screen'
 import LessonNextSteps from './components/LessonNextSteps'
 import LessonData from './components/LessonData'
+import WistiaVideo from './components/WistiaVideo'
 
 export default ({instructor, lesson}) => (
-  <Screen
-    title={lessonTitleText}
-    main={
-      <LessonData 
-        instructor={instructor}
-        lesson={lesson} 
-      />
-    }
-    aside={
-      <LessonNextSteps lesson={lesson} />
-    }
-  />
+  <div>
+    {lesson.wistia_id
+      ? <WistiaVideo wistiaId={lesson.wistia_id}/>
```

Comment:
I'd remove `Screen` from the lesson page entirely as it is likely not going to fit the caged format. I'd be more inclined to add a selection of generic layout structures versus the one-size-fits-all `Screen` approach.